packages/__docs__/buildScripts/build-docs.ts

```diff
@@ -177,9 +177,7 @@ function processSingleFile(fullPath: string) {
     // Components (e.g. Alert) store their descriptions in README.md files.
     // Add this to the final JSON if it's edited
     const readmeDesc = tryParseReadme(dirName)
-    docObject.description = readmeDesc
-      ? docObject.description + readmeDesc
-      : docObject.description
+    docObject.description = readmeDesc ? readmeDesc : docObject.description
```

Comment:
The problem is that we just changed it to this, because if we don't, the TS types won't appear on the main component (that has it's own readme file). We must find a way for the docs app the pick up the flags we put there that have some effect on the docs render.

![image](https://user-images.githubusercontent.com/11446033/162405254-3c553f72-7eaf-4a62-920b-ffb0cbd8ee7e.png)

Reply:
fixed it. I also deleted the documentation in the comment of `InstUISettingsProvider` and `EmotionThemeProvider` because it was showing both. After the last few components are converted to TS this logic can be simplified: If there is a readme use that for docs, otherwise the comment above the component/function. This way we can add short comments about what components do helping to use InstUI in the IDE.